unlock-app/src/services/storageService.ts

```diff
@@ -225,8 +235,13 @@ export class StorageService extends EventEmitter {
    */
 
   async createUser(user: any) {
-    const opts = {}
-    return axios.post(`${this.host}/users/`, user, opts)
+    return fetch(`${this.host}/users/`, {
+      method: 'POST',
+      body: JSON.stringify(user),
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    })
```

Comment:
Can you check this doesn't actually change the assumption behind the function?

The fetch version will return response while the post version should return the json/text body, I believe.

Reply:
This function is used in 'createUserAccount` in useAccount.ts. I modified that function get the response data using  `response.json()` , so that should be good. I will check the rest to make sure. 
![cua](https://user-images.githubusercontent.com/32372824/179555088-47538dba-1a8e-4aed-8839-0cfb8cb74ee3.png)

Reply:
Is that the only place where it's used?

Reply:
Yes I believe that's the only place it's used!